internal/exec/exec_test.go

```diff
@@ -127,7 +127,7 @@ func TestExecStdinPowershell(t *testing.T) {
 
 func TestExecsWithJob(t *testing.T) {
 	// Test that we can assign processes to a job object at creation time.
-	job, err := jobobject.Create(context.Background(), &jobobject.Options{Name: "test"})
+	job, err := jobobject.Create(context.Background(), nil)
```

Comment:
Made a separate PR to change these. Will rebase if that gets in https://github.com/microsoft/hcsshim/pull/1335/files